tests/Unit/RmStepTest.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+use WordPress\Blueprints\BlueprintException;
+use WordPress\Blueprints\Model\DataClass\RmStep;
+use WordPress\Blueprints\Runner\Step\RmStepRunner;
+use WordPress\Blueprints\Runtime\NativePHPRuntime;
+
+
+beforeEach(function() {
+    $this->step = new RmStepRunner();
+    $documentRoot = sys_get_temp_dir();
+    $this->runtime = new NativePHPRuntime($documentRoot);
+    $this->step->setRuntime($this->runtime);
+});
+
+it('should remove a directory when an absolute path provided', function() {
+    $unresolvedDirToRemove = "dir";
+    $resolvedDirToRemove = $this->runtime->resolvePath($unresolvedDirToRemove);
+    mkdir($resolvedDirToRemove, 0777, true);
+    expect(file_exists($resolvedDirToRemove))->toBeTrue();
+
+    $input = new RmStep();
+    $input->path = $resolvedDirToRemove;
+
+    $this->step->run($input);
+
+    expect(file_exists($resolvedDirToRemove))->toBeFalse();
+});
+
+it('should remove a directory when a relative path provided', function () {
+    $unresolvedDirToRemove = "dir";
+    $resolvedDirToRemove = $this->runtime->resolvePath($unresolvedDirToRemove);
+    mkdir($resolvedDirToRemove, 0777, true);
+    expect(file_exists($resolvedDirToRemove))->toBeTrue();
+
+    $input = new RmStep();
+    $input->path = $unresolvedDirToRemove;
+
+    $this->step->run($input);
+
+    expect(file_exists($resolvedDirToRemove))->toBeFalse();
+});
+
+it ('should remove a directory with a subdirectory', function () {
+    $unresolvedSubDir = "dir/subdir";
+    $resolvedSubDir = $this->runtime->resolvePath($unresolvedSubDir);
+    mkdir($resolvedSubDir, 0777, true);
+    expect(file_exists($resolvedSubDir))->toBeTrue();
+
+    $unresolvedDirToRemove = "dir";
```

Comment:
You could use the `dirname()` PHP function and avoid introducing new variables.

Reply:
I guess it's a matter of taste at this point. Personally I'd just call `dirname()` twice to avoid introducing additional names for things.

Reply:
I acquiesce.